Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccreissuance/core/server/DccReissuanceServer.kt

```diff
@@ -61,16 +62,24 @@ class DccReissuanceServer @Inject constructor(
             return
         }
 
-        when (code()) {
-            400 -> ErrorCode.DCC_RI_400
-            401 -> ErrorCode.DCC_RI_401
-            403 -> ErrorCode.DCC_RI_403
-            406 -> ErrorCode.DCC_RI_406
-            429 -> ErrorCode.DCC_RI_429
-            500 -> ErrorCode.DCC_RI_500
-            in 400..499 -> ErrorCode.DCC_RI_CLIENT_ERR
-            else -> ErrorCode.DCC_RI_SERVER_ERR
-        }.also { throw DccReissuanceException(errorCode = it) }
+        val serverErrorCode = tryGetServerErrorCode()
```

Comment:
I think better naming would be `val severError = tryGetServerError()` ?
Here we are not just getting the code , but the whole `error (code + message)`

Reply:
👌 adjusted